import pytest
from app.services import insurances_service
from app.exceptions.database_errors import UnableToFindIdError
from app.resources.insurance_resource import InsuranceReturnResource

# VALID TESTS FOR get_insurance_by_id

# INVALID TESTS FOR get_insurance_by_id

# VALID TESTS FOR get_all_insurances

# INVALID TESTS FOR get_all_insurances
